lib/stripe/api_version.rb

```diff
@@ -4,6 +4,5 @@
 module Stripe
   module ApiVersion
     CURRENT = "2024-06-20"
-    PREVIEW = "cs_ubb_launch"
```

Comment:
Current API version should be: `2024-09-30.acacia`

Reply:
hm i wonder why this wasn't fixed by codegen

Reply:
Even the rubocop lines aren't getting copied over. 

https://github.com/stripe/sdk-codegen/blob/master/src/ruby/generator.tsx#L747